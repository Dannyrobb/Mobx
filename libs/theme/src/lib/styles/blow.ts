import { CSS } from './';
//
export const blow: CSS = {
  top: 0,
  right: 0,
  bottom: 0,
  left: 0,
};

export const absoluteBlow: CSS = {
  ...blow,
  position: 'absolute',
};

export const fixedBlow: CSS = {
  ...blow,
  position: 'fixed',
};
