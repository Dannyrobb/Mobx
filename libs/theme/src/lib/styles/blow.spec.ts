import { blow } from './blow';

describe('blow', () => {
  it('should render successfully', () => {
    const result = blow;
    expect(result).toMatchSnapshot();
  });
});
