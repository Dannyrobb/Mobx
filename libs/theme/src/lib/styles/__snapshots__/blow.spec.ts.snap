// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blow should render successfully 1`] = `
Object {
  "bottom": 0,
  "left": 0,
  "right": 0,
  "top": 0,
}
`;
